src/Entity/Character.cpp

```diff
@@ -13,39 +13,49 @@ namespace game
 
 void Character::InitCharacter(flecs::world& ecs)
 {
-	//auto q = ecs.query<CollisionComponent, TransformComponent>();
-
 	const TransformComponent transformComponent{Vector2{100, 100}, Vector2Scale(Vector2{CHARACTER_WIDTH, CHARACTER_HEIGHT}, ENTITY_SCALE)};
-	ecs.entity("Player")
+	
+	//QUESTION: do we put const in there, since we would change the components, and not the
+	//entity? Is entity stored as one object in memory, or it has links
+	//to addresses to each component and we cans afely put const here?
+	//(if yes, wouldnt it mean we always can put const?)
```

Comment:
In memory, the entity is only an id. Although it's still handled as a `struct`, meaning we can call member functions defined for `flecs::entity`.

We can only mark it `const` if we're calling `const` member functions. In this case, we don't, so we should mark it `const`.

But if we were to make the following call after, we wouldn't be able to mark it `const`, because `.add<>` is not a `const` member function:
```cpp
playerEntity.add<CharacterTag>();
```